labrad/test/test_server.py

```diff
@@ -0,0 +1,32 @@
+import Queue
+
+import pytest
+
+import labrad
+from labrad.server import LabradServer, setting
+from labrad import util
+
+def test_server_expire_context_method_is_called():
+    """Ensure that server's expireContext method is called when client disconnects."""
+    queue = Queue.Queue()
+
+    class TestServer(LabradServer):
+        name = "TestServer"
+
+        def expireContext(self, c):
+            queue.put(c.ID)
+
+        @setting(100)
+        def echo(self, c, data):
+            return data
+
+    server = TestServer()
+    with util.syncRunServer(server):
+        with labrad.connect() as cxn:
+            context = (cxn.ID, 4)
```

Comment:
It doesn't mean anything; changed this to pick a random context instead to make this clear.